local_adaptation/README.md

```diff
@@ -0,0 +1,28 @@
+##  Local Adaptation NWF
+
+This is a simulation in which local adaptation is achieved through QTL mutations arising in a 20kb region of the 1e6bp genomic element. Neutral mutations may be overlayed for downstream analysis (say, using msprime), as they are excluded from this example. 
```

Comment:
Could add here that this borrow ideas from SLiM recipe XYZ, combining new ideas presented in the rest of this repository.